sdk/tests/wallet/wallet_storage.rs

```diff
@@ -316,6 +316,42 @@ async fn check_existing_db_4() -> Result<()> {
     tear_down(storage_path)
 }
 
+// Db created with iota-sdk commit 37edd0706ee003a0d17c7da19ba974b17b365cfe (@iota/wallet@2.0.2-alpha.32)
+#[cfg(feature = "stronghold")]
+#[tokio::test]
+async fn check_existing_db_5() -> Result<()> {
+    let storage_path = "check_existing_5_db_test";
+    setup(storage_path)?;
+    // Copy db so the original doesn't get modified
+    copy_folder("./tests/wallet/fixtures/check_existing_5_db_test", storage_path).unwrap();
+
+    let wallet = Wallet::builder().with_storage_path(storage_path).finish().await?;
+
+    // Commented because it wasn't created with encrypt_work_factor 0
+    // wallet.set_stronghold_password("STRONGHOLD_PASSWORD".to_owned()).await?;
+
+    // TODO: Supposed to be 2 or was that a mistake?
```

Comment:
?

Reply:
Yes sorry, initially I created a db with two accounts, but then recreated it with one because I changed the paths